### PR TITLE
Add GA logs to help troubleshoot GA not created bug

### PR DESCRIPTION
**Pull request by al-hmcts**
### Jira link

See [DFR-2698](https://tools.hmcts.net/jira/browse/DFR-2698)

### Change description

Add additional logging to Create General Application event. 

### Checklist

- [X] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by dawudgovuk — APPROVED** [review]

**Review by hamzahtahirhmcts — APPROVED** [review]

**Review by jthmcts — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/GeneralApplicationAboutToStartHandler.java [hunk 1]
```diff
@@ -58,14 +58,19 @@ public GenericAboutToStartOrSubmitCallbackResponse<FinremCaseData> handle(Finrem
...
             helper.getGeneralApplicationList(caseData, GENERAL_APPLICATION_COLLECTION);
+
+        log.info("GA AboutToStartHandler beginning existing General Application List size: {} for case ID: {}",
+            existingGeneralApplication.size(), caseId);
+
         GeneralApplicationCollectionData data =
             helper.mapExistingGeneralApplicationToData(caseData, userAuthorisation, caseId);
 
         String loggedInUserCaseRole = assignCaseAccessService.getActiveUser(caseId, userAuthorisation);
-        log.info("Logged in user case role type {} on Case ID: {}", loggedInUserCaseRole, caseId);
```
Comment (ptrelease) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/GeneralApplicationAboutToStartHandler.java:65]: Just checking you want this gone, as I don't see it replaced.

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/GeneralApplicationAboutToStartHandler.java [hunk 2]
```diff
@@ -58,14 +58,19 @@ public GenericAboutToStartOrSubmitCallbackResponse<FinremCaseData> handle(Finrem
...
         caseData.setCurrentUserCaseRoleType(loggedInUserCaseRole);
 
         if (data != null) {
+            log.info("Data found on working model for General Application model, "
+                + "adding to existing General Applications list");
```
Comment (ptrelease) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/GeneralApplicationAboutToStartHandler.java:73]: pop a case id in.

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/GeneralApplicationAboutToStartHandler.java (hunk 1 above)
Comment (ptrelease) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/GeneralApplicationAboutToStartHandler.java:65]: OK.  I see you've done more than once.

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/GeneralApplicationMidHandler.java [hunk 3]
```diff
@@ -125,6 +129,7 @@ public GenericAboutToStartOrSubmitCallbackResponse<FinremCaseData> handle(Finrem
                 }
             }
             case INTERVENER4 -> {
+                log.info("Mapping Intervener4 GAs");
                 generalApplications = wrapper.getIntervener4GeneralApplications();
                 generalApplicationsBefore = wrapperBefore.getIntervener4GeneralApplications();
                 listElement.setCode(INTERVENER4);
```
Comment (ptrelease) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/GeneralApplicationMidHandler.java:135]: You have case ID available to add to the above,  consider adding.

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/GeneralApplicationAboutToStartHandler.java (hunk 1 above)
Comment (ptrelease) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/GeneralApplicationAboutToStartHandler.java:65]: get active user - got it.

**Review by dawudgovuk — APPROVED** [review]

**Review by ptrelease — APPROVED** [review]

**Review by hamzahtahirhmcts — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/GeneralApplicationAboutToStartHandler.java (hunk 1 above)
Comment (al-hmcts) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/GeneralApplicationAboutToStartHandler.java:65]: This is duplicate log, the same log is in the assignCaseAccessService.getActiveUser() method call

**Pull request merged**